Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM rancher/dind:v0.2.0
 COPY ./scripts/bootstrap /scripts/bootstrap
 RUN /scripts/bootstrap
 WORKDIR /source
+VOLUME /target
```

Comment:
@cloudnautique I'm guessing we never have multiple concurrent runs of the validation-tests on the same host?